ha/setup/pcs_config/alert_config.py

```diff
@@ -49,7 +49,8 @@ def create_alert(self):
         """
         Log.info("Creating pacemaker alert ...")
         if self.is_alert_exists():
-            self.delete_alert()
```

Comment:
Why deletion is avoided here ?

Reply:
The earlier logic was to remove an alert if its already present and new alert will get added . This was present since alert config was part of cluster creation.
Now alert creation is moved out of cluster creation and will get executed for each node addition.
So the delete is removed.
@ajay-paratmandali can you confirm the same ?

Reply:
Oh Ok..!